demo-app/tsconfig.json

```diff
@@ -11,7 +11,7 @@
     "moduleResolution": "bundler",
     "resolveJsonModule": true,
     "isolatedModules": true,
-    "jsx": "preserve",
+    "jsx": "react-jsx",
```

Comment:
What's this do @fionaochs?

Reply:
Oops, this snuck in. @jonmagic and I were pairing on this on Friday and were running into JSX errors and so we changed this but determined that the  ```"jsx": "preserve"``` is correct. Ill change it back